src/main/java/com/google/cloud/tools/appengine/api/deploy/DefaultStageStandardConfiguration.java

```diff
@@ -1,157 +0,0 @@
-/*
```

Comment:
I would like for the interfaces to remain in, in some way for the build tools at the moment.

Reply:
Interfaces can't enforce non-nullness. Do the build tools currently implement this interface?

Reply:
yes, to expose it directly to the build config. Oh is it because the annotation doesn't pass through?

Reply:
No, it's not the annotation. An implementation can return anything. A class can enforce preconditions and postconditions. An interface can't.